README.md

```diff
@@ -22,6 +22,57 @@ The code contains four sample backends:
 * p4c-graphs: can be used to generate visual representations of a P4 program;
   for now it only supports generating graphs of top-level control flows.
 
+Sample command lines:
+
+Compile P4_16 or P4_14 source code.  If your program successfully
+compiles, the command will create files with the same base name as the
+P4 program you supplied, and the following suffixes instead of the
+`.p4`:
+
++ a file with suffix `.p4i`, which is the output from running the
```

Comment:
.p4i was the suffix I came up with when designing the p4c driver, but it is really just the output from c preprocessor, should we call it *.i or *.ii to follow the naming convention of C? No big deal, just feel like a waste of good suffix on something not very important.

Reply:
I personally don't mind the "p4i" suffix.  I didn't realize there was a conventional *.i or *.ii suffix for C after preprocessing.